### ISSUE TITLE
 [Feature request] Filter search results by app permissions

### ISSUE THREAD

**Issue by AdKiller** [yeriomin/YalpStore#380]
It would be great if you could search for apps by permission. 

For example, this app used to be able to that: https://play.google.com/store/apps/details?id=com.hasslefixes.stripsearch&hl=en

Thanks

**Comment by IzzySoft** [comment]
As it's no longer listed on Playstore, be welcome to check out an article I wrote on StripSearch about 4 years ago: [StripSearch helps you find apps by permission](https://android.izzysoft.de/articles/named/stripsearch-finds-apps-by-permissions). This is something really needed, and I already got asked for such an app multiple times. Would be great if yalP could offer this functionality.

**Comment by AdKiller** [comment]
This is even more crucial now that Google doesn't want you to see anymore what permissions are requested **before** you download an app from Google Play!

**Comment by IzzySoft** [comment]
Indeed! I've just pinged the dev of StripSearch. He'll offer his help to the Yalp team for this (if it's accepted) – and also see if StripSearch can be revived. The two don't contradict: It will be good to have that functionality directly here in Yalp – and those still using the bloated Playstore app can have this functionality separately via StripSearch. Looking forward to that!

**Comment by IzzySoft** [comment]
@yeriomin any chance for that in Yalp? I meanwhile got response from the StripSearch dev (actually, already a little ago). Unfortunately StripSearch used APIs no longer provided by Google. But he'd still be willing to help.

**Comment by AdKiller** [comment]
@maracuja-juice care to elaborate why this is a bad idea that you gave it a thumbs down?
